x-pack/legacy/plugins/siem/public/pages/detection_engine/create_rule/components/description_step/index.tsx

```diff
@@ -36,16 +48,34 @@ const EuiFlexItemWidth = styled(EuiFlexItem)`
   width: 50%;
 `;
 
+const MyEuiListGroup = styled(EuiListGroup)`
+  padding: 0px;
+  .euiListGroupItem__button {
+    padding: 0px;
+  }
+`;
+
+const ThreatsEuiFlexGroup = styled(EuiFlexGroup)`
+  .euiFlexItem {
+    margin-bottom: 0px;
+  }
+`;
+
 export const StepRuleDescription = memo<StepRuleDescriptionProps>(
   ({ data, indexPatterns, schema }) => {
     const keys = Object.keys(schema);
+    const listItems = keys.reduce(
+      (acc: ListItems[], key: string) => [
+        ...acc,
+        ...buildListItems(data, pick(key, schema), indexPatterns),
+      ],
+      []
+    );
     return (
       <EuiFlexGroup gutterSize="none" direction="row" justifyContent="spaceAround">
-        {chunk(keys.includes('queryBar') ? 3 : Math.ceil(keys.length / 2), keys).map(key => (
+        {chunk(Math.ceil(listItems.length / 2), listItems).map(chunckListItems => (
           <EuiFlexItemWidth grow={false}>
-            <EuiDescriptionList
-              listItems={buildListItems(data, pick(key, schema), indexPatterns)}
-            />
+            <EuiDescriptionList listItems={chunckListItems} />
```

Comment:
Missing a key here -- seeing a react key error in the console after submitting `Define Rule` section.